go.sum

```diff
@@ -640,24 +641,28 @@ github.com/onflow/cadence v0.4.0-beta1/go.mod h1:gaPtSctdMzT5NAoJgzsRuwUkdgRswVH
 github.com/onflow/cadence v0.10.1/go.mod h1:ORAnWydDsrefAUazeD1g+l7vjNwEuJAcZ7bMz1KnSbg=
+github.com/onflow/cadence v0.10.2 h1:uBFhdlp0blYCddZTrnCjbLEVl/aYq1/9iP949KxzfbI=
+github.com/onflow/cadence v0.10.2/go.mod h1:ORAnWydDsrefAUazeD1g+l7vjNwEuJAcZ7bMz1KnSbg=
 github.com/onflow/cadence/languageserver v0.10.1 h1:5PZGTMSW5TTN9rvILBCIytQVnEpQVgyR58EL0RU5QOg=
 github.com/onflow/cadence/languageserver v0.10.1/go.mod h1:n6+K0BglslORHKPr4IOwIUArTYR6YIFe0cFivfID4eQ=
 github.com/onflow/flow-core-contracts/lib/go/contracts v0.1.1-0.20201002123512-35d751ebea1d h1:5DHPRH9rdU93csffCBjhhypyPGnx4nnYsumeynUEltk=
 github.com/onflow/flow-core-contracts/lib/go/contracts v0.1.1-0.20201002123512-35d751ebea1d/go.mod h1:yuFiT2+dZm42smG7XZQlMgZyb31hn5dvLrIDq0/PVc8=
-github.com/onflow/flow-emulator v0.12.1 h1:mg0vsiUllWnvEL4fTcgxCdUxrCTlzGGD6q69sDDVjfI=
-github.com/onflow/flow-emulator v0.12.1/go.mod h1:GH6VMgo3v3mf2c09YKKfSwzcC+DZG5ZbEY2vO1SwRls=
+github.com/onflow/flow-emulator v0.12.2 h1:xolCxQq7rZZ5xF8sgh9kHFcsEYtd33NlOUMR25Jhze0=
+github.com/onflow/flow-emulator v0.12.2/go.mod h1:+AdB0qLyGXHaugJExb5drE0wjoS7s2fBlbhBDD+N25Q=
 github.com/onflow/flow-ft/contracts v0.1.3/go.mod h1:IKe3yEurEKpg/J15q5WBlHkuMmt1iRECSHgnIa1gvRw=
 github.com/onflow/flow-ft/lib/go/contracts v0.2.1-0.20201002112420-010719813062 h1:atzsWTUg5NEavUlIv48R+3a6WPvEHu5Prd5FQUYDLaM=
 github.com/onflow/flow-ft/lib/go/contracts v0.2.1-0.20201002112420-010719813062/go.mod h1:Mf//HSOJUSzr4Z7QmW4BZGSweoqlrJJJ92KvKnqGGQk=
-github.com/onflow/flow-go v0.11.1-0.20201109210205-312549711f62 h1:jDAQ/9zWx5FvdTppEkB4DkhqlrJ1ojN42UeyYJRijgs=
-github.com/onflow/flow-go v0.11.1-0.20201109210205-312549711f62/go.mod h1:PAyG7Z8v39fE+0h4OKQBJiD+D1GA2TCNwgnFFKNnqjs=
+github.com/onflow/flow-go v0.12.0 h1:Utrd4bjFPVhN6szV/4Gkdd2QYgOG+nRvamkp6Pdkcyc=
+github.com/onflow/flow-go v0.12.0/go.mod h1:Oq6P/56vrJGlB7sQR/R/JlAQwwyP9qmxVG9WS+yfkjs=
 github.com/onflow/flow-go-sdk v0.4.0/go.mod h1:MHn8oQCkBNcl2rXdYSm9VYYK4ogwEpyrdM/XK/czdlM=
 github.com/onflow/flow-go-sdk v0.12.1 h1:tdY1wxECl15CKSA5jJ6lBtDU/iISQRn91DsEBulYtew=
 github.com/onflow/flow-go-sdk v0.12.1/go.mod h1:7sVH3rpQ+zk2I2xLF/Ob/TCOa3Mz+N2J5DtCDtLBtv4=
+github.com/onflow/flow-go-sdk v0.12.2 h1:ukIAek19pwYoHmSSYeaJ729vCA+04VEv6tCRlxwiWRI=
+github.com/onflow/flow-go-sdk v0.12.2/go.mod h1:jWwkpCkhxzeGmCs7oV6ydx50svez/PYUuhg8Y7dY0Ko=
 github.com/onflow/flow-go/crypto v0.9.4 h1:jSMo91fNCF5Dywt0VEM/AEf/O1fcqRYvubezi9AhlaU=
 github.com/onflow/flow-go/crypto v0.9.4/go.mod h1:oXuvU0Dr4lHKgye6nHEFbBXIWNv+dBQUzoVW5Go38+o=
-github.com/onflow/flow-go/crypto v0.11.0 h1:LsyrIqYFu/THD4tSiCuKq9h/woMofdwvR6r45ZdVYrM=
-github.com/onflow/flow-go/crypto v0.11.0/go.mod h1:oXuvU0Dr4lHKgye6nHEFbBXIWNv+dBQUzoVW5Go38+o=
+github.com/onflow/flow-go/crypto v0.11.1-0.20201124074740-4553dbb0bc38 h1:GIKNc6pq8sBuhY6IxYkXptrs3UQhGvxlrmj/+xEywLA=
+github.com/onflow/flow-go/crypto v0.11.1-0.20201124074740-4553dbb0bc38/go.mod h1:oXuvU0Dr4lHKgye6nHEFbBXIWNv+dBQUzoVW5Go38+o=
```

Comment:
Should we point to the same `crypto` version as `flow-go` version?

Reply:
Ideally we would us the version of the crypto module that flow-go uses.

However, the version of crypto pinned in flow-go is wrong (see https://github.com/onflow/flow-go/pull/137, brought it up yesterday again: https://axiomzen.slack.com/archives/C015G65HR2P/p1606232145256900). There's a tag for crypto/v0.12.0 missing, which flow-go should use.

This bump is to crypto as of flow-go tag v0.12.0, which is what flow-go implicitly uses when its built due to the replace statements in its go.mod file. This is basically just a workaround for upstream problem.

Reply:
Right. I mean if we specify the same version/commit here (both `v0.12.0`), it will ensure the build uses both modules at the same version. Unless `flow-go` is somehow pinning a version of `flow-go/crypto` that is _after_ its own version, the version specified here will be chosen.

But I agree the proper solution is removing the `replace` in `flow-go`

Reply:
`flow-go/crypto` is it's own module, and in Go, must be tagged separately, as `crypto/v.012.0`, but this tag is missing; the tag `v0.12.0` without the nested module name is not considered. So it's basically using the commit that should have the tag.